examples/chat/module_1/rust/src/lib.rs

```diff
@@ -14,79 +14,76 @@
 // limitations under the License.
 //
 
-use chat_common::proto::chat::ReceivedMessage;
-use log::{info, warn};
+use chat_common::command::Command;
+use chat_common::proto::chat::Message;
+use log::{error, info};
 use protobuf::ProtobufEnum;
 
-#[cfg(target_arch = "wasm32")]
 #[no_mangle]
 pub extern "C" fn oak_main(handle: u64) -> i32 {
-    match std::panic::catch_unwind(|| main(handle)) {
-        Ok(rc) => rc,
-        Err(_) => oak::OakStatus::ERR_INTERNAL.value(),
-    }
+    std::panic::catch_unwind(|| {
+        let room = Room::default();
+        room.event_loop(handle)
+    })
+    .unwrap_or_else(|_| oak::OakStatus::ERR_INTERNAL.value())
 }
 
-pub fn main(in_handle: u64) -> i32 {
-    oak_log::init_default();
-    oak::set_panic_hook();
```

Comment:
Have these two lines been lost along the way (and with it the ability to log from the chat frontend Node)?

Reply:
Good catch, added back to `oak_main`.